mars/services/meta/web.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2020 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from mars.services.web.core import ServiceWebHandlerBase, ServiceWebAPIBase
+from tornado.httpclient import AsyncHTTPClient
+from .api import MetaAPI
+
+
+class MetaWebHandler(ServiceWebHandlerBase):
+    _api_cls = MetaAPI
+
+    async def create(self, session_id: str, address: str):
+        api_instance = await MetaAPI.create(session_id, address)
+        self._api_instances[id(api_instance)] = api_instance
+        return id(api_instance)
+
+    async def create_session(self, session_id: str, address: str):
```

Comment:
What's the difference between `create` and `create_session`? These two methods are the same.

Reply:
This is a typo, `create_session` should invoke `create_session` of `MetaAPI`